registry_quickstart/developers.adoc

```diff
@@ -80,3 +80,75 @@ stored in `/root/.docker/config.json`. Without *sudo*, Docker login credentials
 *root* user. This would cause docker commands to fail authentication if *sudo*
 is used inconsistently.
 ====
+
+=== Using Service Account Tokens for Docker Login
+
+For long-lived, token-based authentication users may create
+link:../admin_guide/service_accounts.html[service account tokens] to
+authenticate with docker. This is particularly useful when integrating automation.
+Service accounts must be configured using the CLI. See
+link:../cli_reference/get_started_cli.html[getting started with the CLI].
```

Comment:
xref:../cli_reference/get_started_cli.adoc#cli-reference-get-started-cli[getting started with the CLI]